locales/en/messages.json

```
@@ -4858,6 +4858,13 @@
     "osdDescElementVtxChannel": {
         "message": "Current VTX channel and power"
     },
+    "osdTextElementVTXchannelVariant-power": {
+        "message": "VTX power",
+        "description": "One of the elements of the OSD"
+    },
+    "osdTextElementVTXchannelVariant-full": {
+        "message": "Band:Channel:Pwr:Pit  "
```

Add the description to be aligned with the other. The description is used by the translators.
```suggestion
        "message": "Band:Channel:Pwr:Pit",
        "description": "One of the variants of the altitude element of the OSD"
```